internal/queryexecutor/execute_conditions.go

```diff
@@ -151,6 +153,32 @@ func (e *WorldStateJSONQueryExecutor) execute(dbName string, attribute string, c
 			return nil, err
 		}
 
+		if plan.seekKey == nil || seeked {
+			keys[indexEntry.Key] = true
+			continue
+		}
+
+		if indexEntry.Value != plan.seekKey.Value {
+			keys[indexEntry.Key] = true
+			continue
+		}
+
+		seekKey, err := plan.seekKey.String()
+		if err != nil {
+			return nil, err
+		}
+		itemExist := iter.Seek([]byte(seekKey))
+		seeked = true
+
+		if !itemExist {
+			break
+		}
```

Comment:
Do we expect this to ever match? keys in the index have always Existing/`v`/Existing, so if i understand correctly, since we are seeking Existing/`v`/Ending, this will get the the next value after `v`, no?
If so, why not just continue to iterate?